kratos/includes/io.h

```diff
@@ -108,16 +108,20 @@ class KRATOS_API(KRATOS_CORE) IO
     ///@{
 
     /// Default constructor.
-    IO() {}
+    IO() = default;
 
     /// Destructor.
-    virtual ~IO() {}
+    virtual ~IO() = default;
 
+    /// Copy constructor.
+    IO(IO const& rOther) = delete;
```

Comment:
FYI deleting it in the baseclass propagates to derived classes (unless they define their own)

Reply:
Good to know.